modules/reference/pages/systems/system-details/sd-sp-migration.adoc

```diff
@@ -1,7 +1,8 @@
 [[ref-systems-sd-migration]]
-= SP Migration
```

Comment:
Same remark about the file name as above

Reply:
yes, I'll do it.   I'll commit it separately.